### PR TITLE
Add KeyderJS library.

### PR DESCRIPTION
**Pull request by JacquesMarais**

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
Please take a look at our document for what you need to send a PR, thanks.

**Comment by PeterDaveHello** [comment]
You should use only one commit to add this lib, add minified file, and fix the invalid package.json, thanks.

**Comment by PeterDaveHello** [comment]
No response?

**Comment by JacquesMarais** [comment]
Sorry, I just can't figure out how to do this thing. I have never done it before.

**Comment by PeterDaveHello** [comment]
You can use `git rebase` to squash these commits into only one commit and fix the error in package.json

Reference:
https://help.github.com/articles/about-git-rebase/
http://git-scm.com/book/en/v2/Git-Tools-Rewriting-History

File: ajax/libs/KeyderJS/package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  "filename": "keyder.js",
+  "version": "1.0.0",
+  "description": "KeyderJS is a simple, yet powerful, JavaScript event library.",
+  "homepage": "https://github.com/GethisOrg/KeyderJS/",
+  "keywords": [
+    "keyder"
+    "events",
+    "javascript",
+    "library",
+    "event",
```
Comment (PeterDaveHello) [inline ajax/libs/KeyderJS/package.json]: Should not have comma here

File: ajax/libs/KeyderJS/package.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "KeyderJS",
+  "filename": "keyder.js",
+  "version": "1.0.0",
+  "description": "KeyderJS is a simple, yet powerful, JavaScript event library.",
+  "homepage": "https://github.com/GethisOrg/KeyderJS/",
+  "keywords": [
+    "keyder"
```
Comment (PeterDaveHello) [inline ajax/libs/KeyderJS/package.json]: Should have comma here

**Pull request closed without merging**